config/settings/loadtest.yml

```diff
@@ -1,18 +1,28 @@
 gcp_api_key: please_change_me
 publish_api_url: https://publish-teacher-training-loadtest.london.cloudapps.digital
-publish_url: https://qa.publish-teacher-training-courses.service.gov.uk
-find_url: https://qa.find-postgraduate-teacher-training.service.gov.uk
+publish_url: https://publish-loadtest.london.cloudapps.digital
+find_url: https://find-loadtest.london.cloudapps.digital
 
 # URL of this app for the callback after sigining in
-base_url: https://publish-teacher-training-loadtest.london.cloudapps.digital
+base_url: https://publish-loadtest.london.cloudapps.digital
+
+search_ui:
+  base_url: https://find-loadtest.london.cloudapps.digital
+
+dfe_signin:
```

Comment:
We agreed to now use DFE sign-in so can we remove this?

Reply:
Removed DfE Sign-in settings